### ISSUE TITLE
Disable the opening of a new window when editing another file

### ISSUE THREAD

**Issue by wilfryed** [adobe/brackets#11248]
On Windows 8.1, each time I open a file on Brackets it open a new instance of Brackets. So if you select ten files and select "open with Brackets", you got ten windows of Brackets!
![issue](https://cloud.githubusercontent.com/assets/5651175/8102928/57fccc46-1024-11e5-853b-4fa906aab6db.png)

**Comment by nethip** [comment]
I do not have a Windows 8.1 system to test this. @abose Will you be able to test this and let me know if could repro this?

**Comment by abose** [comment]
@nethip looks like the problem is not present if you have a brackets window already open; 
First close all the brackets instances, and then select a couple of files in explorer, and then right click and chose open with brackets. There will be multiple windows open.

**Comment by nethip** [comment]
@abose Thanks for testing this! My understand from the test results is that, this necessarily is not tied to 8.1. Right?

**Comment by abose** [comment]
Yes, I think so. It might be that as many brackets windows are opened at the same time; and none of the brackets bootup has finished, the system couldn't figure out the active brackets window and ends up opening several windows thinking that no `brackets` window is active.
